File: Programmers/Lv.0/문자열 밀기/120921.java
class Solution {
    public int solution(String A, String B) {
        int answer = -1;
        
        int l = A.length();
        int cnt = 0;
        for (int i=0; i<l; i++) {
            if (A.equals(B)) {
                answer = cnt;
                break;
            }
            
            A = A.substring(l-1, l) + A.substring(0, l-1);
            cnt++;
        }
        
        return answer;
    }
}

File: Programmers/Lv.0/문자열 밀기/120921.js
function solution(A, B) {
    let answer = -1;
    
    const l = A.length;
    let cnt = 0;
    for (let i=0; i<l; i++) {
        if (A === B) {
            answer = cnt;
            break;
        }
        
        A = A.substr(l-1, l) + A.substr(0, l-1);
        cnt++;
    }
    
    return answer;
}
